content/governance.md

```diff
@@ -95,6 +94,12 @@ Decisions are usually made by [lazy consensus](#consensus). If no consensus can
 
 Material changes to this document are discussed publicly on the [developer mailing mailing list][devs]. Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#consensus) unless challenged.
 
+### Editorial changes
+
+Editorial changes are changes which fix spelling or grammar, update work affilition, or similar; they update style or reflect an outside and obvious reality.
+They do not change the intention or meaning of anything in this document.
+They must be made via PR and accepted via [lazy consensus](#consensus).
```

Comment:
Could this prevent a majority vote on these?

Reply:
are we follwing rfc2119 ?
1. MUST   This word, or the terms "REQUIRED" or "SHALL", mean that the
   definition is an absolute requirement of the specification.

Reply:
@brian-brazil actually it is also said: Editorial changes *may* be made by lazy consensus unless challenged.

Reply:
I do think the challenge part is important because whether something is editorial or not may be subjective.

Reply:
@brancz I do not understand your concern. The challenge part is covered by the lazy consensus rules.

Reply:
And the challenged part is also present in this new document, 3 lines above. I was quoting it.

Reply:
Sorry it seems I updated my comment before you submitted yours. We are in agreement.